lib/skywalking/client.lua

```diff
@@ -72,110 +60,54 @@ function Client:startBackendTimer(backend_http_uri)
     end
 end
 
--- Register service
-function Client:registerService(metadata_buffer, backend_http_uri)
+function Client:reportServiceInstance(metadata_buffer, backend_http_uri)
     local log = ngx.log
     local DEBUG = ngx.DEBUG
     local ERR = ngx.ERR
 
     local serviceName = metadata_buffer:get('serviceName')
+    local serviceInstanceName = metadata_buffer:get('serviceInstanceName')
 
     local cjson = require('cjson')
-    local serviceRegister = require("register").newServiceRegister(serviceName)
-    local serviceRegisterParam = cjson.encode(serviceRegister)
+    local reportInstance = require("management").newReportInstanceProperties(serviceName, serviceInstanceName)
+    local reportInstanceParam = cjson.encode(reportInstance)
```

Comment:
it may fail, we need to capture the error message.

`local data, err = json.encode(...)`